### ISSUE TITLE
🐛[BUG]:  Validation Loss Not Converging for Bracket Example

### ISSUE THREAD

**Issue by ghasemiAb** [NVIDIA/physicsnemo-sym#228]
### Version

24.12

### On which installation method(s) does this occur?

Docker

### Describe the issue

Issue: Validation Loss Not Converging for Bracket Example
We are encountering issues with the validation loss not converging during training for the bracket example. Specifically, we tested two different modulus containers, 24.12 and 23.08, and observed different outcomes:

**Container Version 24.12:**
        The validation loss remained unchanged even after running for 2 million iterations. This suggests that the model is not making progress toward convergence.

![Image](https://github.com/user-attachments/assets/f65ff429-405b-4fef-9af5-07f15e390f7f)

**Container Version 23.08:**
        Without restarting from the checkpoint, the results showed some improvement. However, the model did not fully converge, especially for the Z components. After 2 million iterations, the Z components reached a plateau at a validation loss of 0.7, while the other components reached 0.3.
        With restart from the checkpoint, the loss curve started either oscillating or diverging, preventing further progress.

![Image](https://github.com/user-attachments/assets/1b92f9a8-b62e-4d3d-84d8-007330cb3198)
![Image](https://github.com/user-attachments/assets/07290344-25b6-4647-9743-e39ede16dd6b)

**Container Version 23.05:**
        The results with this version were similar to those obtained with container version 23.08, showing a consistent pattern of incomplete convergence, particularly for the Z components.

**ldc_2d_zeroEq with container version  24.12**

validation loss are converging and it seems that convergency problem is not valid for ldc_2d_zeroEqexample

### Minimum reproducible example

```shell

```

### Relevant log output

```shell

```

### Environment details

```shell

```

**Comment by oldschool24** [comment]
I had the same issue when using the PyPi installation. I also struggled with validation loss in the fuselage_panel example. According to the commit history of the example folder, it was uploaded on May 9th, 2023, so I suppose that the results from the documentation were implemented using container version 22.09 or earlier
